spyder/widgets/tests/test_pathmanager.py

```diff
@@ -60,6 +62,58 @@ def test_check_uncheck_path(qtbot):
     assert pathmanager.not_active_pathlist == []
 
 
+def test_synchronize_with_PYTHONPATH(qtbot, mocker):
+    if os.name != 'nt':
+        return
```

Comment:
The way to skip tests is by using `pytest.mark.skipif`, e.g.

https://github.com/spyder-ide/spyder/blob/3.x/spyder/app/tests/test_mainwindow.py#L136